darts/models/forecasting/baselines.py

```diff
@@ -196,15 +196,49 @@ def fit(
 
         return self
 
+    def predict(
```

Comment:
as mentioned in the other comment, we can generalize predict() and move it back to the EnsembleModel